### PR TITLE
How to resize disks document

### PR DESCRIPTION
**Pull request by s-oniel**

### CODE REVIEW

File: articles/azure/azs-how-resize-disks.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+    For more detailed instructions, see the [*Getting Started Guide for UKCloud for Microsoft Azure*](azs-gs.md).
+
+2. In the favourites panel, select **Virtual machines**.
+
+    ![Virtual machines in the favourites panel](images/azsp_vmsmenu.png)
+
+3. In the *Virtual machines* blade, select the virtual machine that the disk is attached to.
+
+    ![VM Disks button](images/azs-browser-button-vm-disks.png)
```
Comment (suehighmoor) [inline articles/azure/azs-how-resize-disks.md]: May be missing something subtle, but this image looks the same as the one below for step 4

File: articles/azure/azs-how-resize-disks.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+***
+
+6. When you're done, click **Save**.
+
+    ![VM disks save button](images/azs-browser-vm-disk-save-button.png)
+
+7. You can monitor the progress of the disk resizing by clicking the **Notifications** icon.
+
+   ![Notification showing updating virtual machine disk](images/azs-update-deployment-progress.png)
```
Comment (suehighmoor) [inline articles/azure/azs-how-resize-disks.md]: Image seems to be broken

File: articles/azure/azs-how-resize-disks.md (hunk 1 above)
Comment (BaileyLawson) [reply articles/azure/azs-how-resize-disks.md]: Removed it, the step itself is fairly self explanatory as well as difficult to capture the step in a screenshot

File: articles/azure/azs-how-resize-disks.md (hunk 2 above)
Comment (BaileyLawson) [reply articles/azure/azs-how-resize-disks.md]: Fixed the image link

**Pull request merged**